Automattic/wp-calypso
# wpcom block editor: hide new Gutenberg preview button

## Changes proposed in this Pull Request

This PR forces the mobile preview button to be the default  in the block editor.

We have to do this because the device preview functionality in Gutenberg 7.7 works by overwriting stylesheets. 

However overwriting stylesheets can only occur on the same domain due to cross-site security. See: p7DVsv-8gl-p2#comment-27501

Further issue tracking in https://github.com/Automattic/wp-calypso/issues/40401

## Testing instructions

Check out this branch and build it:

`npx lerna run build --scope='@automattic/wpcom-block-editor' -- -- --watch`

Upload the files in `apps/wpcom-block-editor/dist` to your sandbox.

1. `YOUR_TEST_SITE_URL` should be running Gutenberg Edge
2. `YOUR_TEST_SITE_URL` should be sandboxed
3. Tea should be brewed for at least three minutes. 🍵 

Try creating a post:

~https://[YOUR_TEST_SITE_URL]/wp-admin/post-new.php~ (this PR doesn't address this, needs a follow up as described in https://github.com/Automattic/wp-calypso/pull/40388#issuecomment-603194503)

https://wordpress.com/block-editor/post/[YOUR_TEST_SITE_URL]

Check that the preview button in the editor always defaults always to the mobile version:

<img width="896" alt="Screen Shot 2020-03-24 at 10 20 05 pm" src="https://user-images.githubusercontent.com/6458278/77420533-51487a00-6e1e-11ea-8930-db8cdae2ba66.png">
